cmd/zc_enumerator.go

```diff
@@ -179,6 +185,25 @@ func newStoredObject(morpher objectMorpher, name string, relativePath string, lm
 	return obj
 }
 
+func newForcedErrorStoredObject(err, name, relativePath, containerName string) storedObject {
+	so := newStoredObject(
+		nil, // No need for a morpher on a intended failure
+		name,
+		relativePath,
+		time.Now(),
+		0,                        // A intended failure has no size,
+		emptyPropertiesAdapter{}, // and no content properties
```

Comment:
Probably best to use the existing constants defined for this purpose, namely: `cmd.noContentProps`and `cmd.noBlobProps` and `cmd.noMetadata`